### ISSUE TITLE
Images of every addon for documentation and showcase

### ISSUE THREAD

**Issue by Hans5958** [ScratchAddons/ScratchAddons#364]
How about we take screenshots of all the addons, so we can either showcase it on the website or document it on the docs? Would be a wasted potential if we only have screenshots of the messaging addon and nothing else.

For example, [the README of Refined GitHub](https://www.github.com/sindresorhus/refined-github/tree/master/readme.md). (This doesn't necessarily that we need to do it on our README)

**Comment by JeffaloBob** [comment]
i was thinking of taking a short video clip tutorial of each addon, maybe that could be Included

**Comment by Hans5958** [comment]
Yeah, if we do it, we will make the production of the video easier.

**Comment by TheColaber** [comment]
Updates?

@WorldLanguages created a GIF here: https://github.com/WorldLanguages/animatedThumbnailsBookmarklet/blob/master/gif.gif
@WorldLanguages what did you use to make it?

**Comment by Hans5958** [comment]
Pretty slow. I only did three for the highlights on the browser, but then there's a lot more.

**Comment by TheColaber** [comment]
I've been doing a tiny bit...
https://github.com/TheColaber/Wiki/blob/master/Addon-usage.md

**Comment by Hans5958** [comment]
About the matter of how to use it, yeah, it could be in docs. But then, this is about images of each addon.

**Comment by mxmou** [comment]
@Hans5958 This could probably be closed because there is ScratchAddons/website#61 now.

**Comment by Hans5958** [comment]
That issue is strictly for the showcase on the website. Thought that also counts, I believe there will be images that has a different format and could give more comprehensive overview and explanation about a certain addon, so that is made as an seperate issue.

**Comment by RedGuy12** [comment]
duplicate of #475 then

**Comment by TheColaber** [comment]
Or is it the other way around? 🤔

**Comment by WorldLanguages** [comment]
lol

**Comment by ChrisIsBadAtFnF** [comment]
I could probably do some of the newer addons.
